### DISCUSSION TITLE
Build with single quotes

### DISCUSSION THREAD

**Discussion by rtritto** [rolldown/rolldown#2401] [Q&A]
What to use for build with single quotes instead of double quotes?

**Comment by Boshen** [comment]
May I ask why do you need to change quotes for the build output? Are there similar options in esbuild or rollup?

**Reply by IWANABETHATGUY** [reply]
No, they dont have. 
But for rollup, it using string replacing method so it could keep original code quote style.

**Comment by rtritto** [comment]
Nvm, probably this feature is useless
